### ISSUE TITLE
Sometime callbacks on testnet not working

### ISSUE THREAD

**Issue by noxonsu** [oraclize/ethereum-api#4]
Hi! Sometime callbacks on testnet not working:

See contract https://testnet.etherscan.io/address/0xb22cd5f9e5f0d62d47e52110d9eec3a45be54498

![](http://dl3.joxi.net/drive/2017/02/27/0004/2038/272374/74/ff72baa960.jpg)

**Comment by bertani** [comment]
Hi, unfortunately in the last couple of days the ropsten testnet has experienced lot of issues due to an "attack" driving the block gas limit to 9 Billion..

As a consequence, most nodes stopped working and the state is bloated with spam transactions.
Gavin Wood proposed first to switch to a "ropsten-softfork", and later to rollback the chain to a clean state via another dedicated softfork ("ropsten-rollback"). This was being discussed mostly here: https://gitter.im/ethcore/parity

We have moved, as most of the community (for example testnet.etherscan.io is now using that one too), to the ropsten-softfork - so Oraclize is now working there, but depending on the ropsten fork (there are 3 now!) on which you were, there could have been dragons.
The situation should be solved in few days, so my suggestion is to stand by and wait for the network to become healthy again and to choose the softfork the community will follow.

To recap, Oraclize is now listening on "ropsten-softfork", we will switch to "ropsten-rollback" eventually - but things are still being discussed. Some old queries might have been impossible to process due to such situation and very high gas limits. If you need to test Oraclize-based contracts, don't forget that you can do it on any test chain via the [ethereum-bridge](https://github.com/oraclize/ethereum-bridge) tool (even truffle/testrpc), or straight in [browser-solidity](http://dapps.oraclize.it/browser-solidity/).
